modules/tensor_mechanics/include/materials/ADLAROMANCEStressUpdateBase.h

```diff
@@ -15,7 +15,8 @@ enum class ROMInputTransform
 {
   LINEAR,
   LOG,
-  EXP
+  EXP,
+  INVERSE
```

Comment:
In working with Aaron on the previous P91 ROM, I learned that the windows for the Inverse transform are switched, e.g. the low limit of effective strain (0.0) became the high window limit (+1) and the high limit of effective strain (~0.02 as an example) became the low window limit (-1). 

We'd need to add a case check to your convertValues method call on line 752 of ADLAROMANCEStressUpdateBase.C to switch the window limits for the inverse case. If you want to drop the INVERSE case for now (it's not being used in any LAROMANCE models that I know of), and add it later after we have time to think through an appropriate regression test, that would be fine with me

Reply:
Whoa, that is way confusing...I'll double check

Reply:
I'm not too sure if this is right or not. Since we don't need it for any of the current implementations, I'll drop it for now. We can circle back if we need to.